elements/rh-back-to-top/test/rh-back-to-top.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { expect, html, nextFrame } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { setViewport, sendKeys } from '@web/test-runner-commands';
+import { type A11yTreeSnapshot, a11ySnapshot } from '@patternfly/pfe-tools/test/a11y-snapshot.js';
+
+import { allUpdates } from '@patternfly/pfe-tools/test/utils.js';
+
+import { RhBackToTop } from '@rhds/elements/rh-back-to-top/rh-back-to-top.js';
+
+const takeProps = (props: string[]) => (obj: object) =>
+  Object.fromEntries(Object.entries(obj).filter(([k]) => props.includes(k)));
+
+describe('<rh-back-to-top>', function() {
+  describe('simply instantiating', function() {
+    let element: RhBackToTop;
+    it('should upgrade', async function() {
+      element = await createFixture<RhBackToTop>(html`<rh-back-to-top></rh-back-to-top>`);
+      const klass = customElements.get('rh-back-to-top');
+      expect(element)
+        .to.be.an.instanceOf(klass)
+        .and
+        .to.be.an.instanceOf(RhBackToTop);
+    });
+  });
+
+  describe('when rendered in a viewport with a height smaller then content length', function() {
+    let element: RhBackToTop;
+    let snapshot: A11yTreeSnapshot;
+
+    beforeEach(async function() {
+      await setViewport({ width: 320, height: 640 });
```

Comment:
Well done

Reply:
my copy and paste skills from my previously written tests in pfe are unmatched 🙃